links.html

```diff
@@ -5,9 +5,9 @@
     <link rel="stylesheet" href="styles/main.css">
   
     <style>
-        main section ul a{
-        color:purple;
-    }
+        #useful-links a{
+            color:purple;
+        }
```

Comment:
NOTE (no marks lost): What you have works for now, but what if you add another section?  Using the id selector will pick specifically the section with that id.